src/plugins/intel_gpu/tests/unit/shape_infer/reorg_yolo_si_test.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "test_utils.h"
+
+#include <intel_gpu/primitives/input_layout.hpp>
+#include <intel_gpu/primitives/reorg_yolo.hpp>
+#include <intel_gpu/primitives/data.hpp>
+
+#include "reorg_yolo_inst.h"
+
+#include "program_wrapper.h"
+
+#include <cmath>
+#include <algorithm>
+
+using namespace cldnn;
+using namespace ::tests;
+
+namespace shape_infer_tests {
+
+struct reorg_yolo_test_params {
+    layout in_layout;
+    uint32_t stride;
+    layout expected_layout;
+};
+
+class reorg_yolo_test : public testing::TestWithParam<reorg_yolo_test_params> { };
+
+TEST_P(reorg_yolo_test, shape_infer) {
+    auto p = GetParam();
```

Comment:
```suggestion
    auto& p = GetParam();
```
To avoid copy of test parameters, unless it is intentional.